projects/Mallard/src/components/front/index.tsx

```diff
@@ -21,19 +21,21 @@ import {
 import { useFrontsResponse } from 'src/hooks/use-issue'
 import { ArticleNavigator } from '../../screens/article-screen'
 import { WithArticle, getAppearancePillar } from '../../hooks/use-article'
+import { WithBreakpoints } from 'src/components/layout/ui/with-breakpoints'
 
 const CollectionPageInFront = ({
     index,
     pillar,
     scrollX,
+    width,
     ...collectionPageProps
 }: {
     index: number
     pillar: PillarFromPalette
     scrollX: Animated.Value
+    width: number
```

Comment:
Could potentially use a `useWidth` hook here that's listens for `change` events? But this is fine too.